### PR TITLE
B2b account

### PR DESCRIPTION
**Pull request by mjasrotia**
Changed the B2B Account schemas as per recent feedback from MARKETO

### CODE REVIEW

File: components/mixins/account/account-details.schema.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  "title": "Account Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/class/account"],
+  "description": "An XDM Business Account.",
+  "definitions": {
+    "account-details": {
+      "properties": {
+        "xdm:accountComponents": {
```
Comment (cmathis) [inline components/mixins/account/account-details.schema.json:19]: If this is an array of b2b-account objects, then why not have the "items" simply be a $ref to that datatype? Is there a reason to duplicate those things here?

Comment (mjasrotia) [reply components/mixins/account/account-details.schema.json:19]: I also thought to refer that initially but I believe, there are still few fields which are not included in the array that exist in the "b2b-account" data type. Let me double check again.

Comment (mjasrotia) [reply components/mixins/account/account-details.schema.json:19]: I think, we can refer that data type in the mixin. I just checked. Thanks for pointing this out. I will make the change.

File: components/datatypes/account-organization.schema.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+          "title": "NAICS Description",
+          "description": "A brief description of an organization’s line of business, based on its NAICS code.",
+          "type": "string"
+        },
+        "xdm:SICDescription": {
+          "title": "SIC Description",
+          "description": "A brief description of an organization’s line of business, based on its SIC code.",
+          "type": "string"
+        },
+        "xdm:D-U-N-S-Number": {
```
Comment (cmathis) [inline components/datatypes/account-organization.schema.json:103]: Cannot use dashes in AEP

File: components/classes/account.schema.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/context/account",
+  "$id": "https://ns.adobe.com/xdm/class/account",
```
Comment (cmathis) [inline components/classes/account.schema.json:8]: You cannot change the $id otherwise you will break any existing schemas that referenced the old $id

Comment (mjasrotia) [reply components/classes/account.schema.json:8]: None of the account schemas are in use yet. All are experimental and we are the only ones changes these schemas.

Comment (mjasrotia) [reply components/classes/account.schema.json:8]: Will change

File: components/datatypes/account-organization.schema.json (hunk 2 above)
Comment (mjasrotia) [reply components/datatypes/account-organization.schema.json:103]: Will remove Dashes

**Review by cmathis — APPROVED** [review]

**Pull request merged**